QUALITY_DECLARATION.md

```diff
@@ -1,20 +1,22 @@
-This document is a declaration of software quality for the `rcutils` package, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+This document is a declaration of software quality for the `rcutils` package, based on the guidelines in [REP-2004](https://github.com/ros-infrastructure/rep/blob/rep-2004/rep-2004.rst).
 
 # `rcutils` Quality Declaration
 
-The package `rcutils` claims to be in the **Quality Level 1** category.
+The package `rcutils` claims to be in the **Quality Level 4** category.
 
-Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Requirements for Quality Level 1 in REP-2004](https://www.ros.org/reps/rep-2004.html).
+Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Requirements for Quality Level 1 in REP-2004 ](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#quality-level-1) of the ROS2 developer guide.
 
 ## Version Policy
 
 ### Version Scheme
+(Missing part, bumping to version 1.0.0)
```

Comment:
Instead of TODOs or comments in the doc. I suggest to add a column with the QD that we are targeting

Reply:
I didn't like the idea of the "TODOs" as part of the current status table because it does reflect something aspirational, and not the current state of the quality declaration. But I liked your idea of summarizing it somewhere to avoid browsing the files to look for TODOs. I opened a new section where all those are listed, with the proper index section. Let me know your thoughts!

Reply:
I think for now it's enough that each package has an issue tracking what's missing for level 1. The declaration is meant to declare facts, I think we can leave the aspirational part out of it.